src/rsa/signing.rs

```diff
@@ -219,6 +217,20 @@ impl RSAKeyPair {
     pub fn public_modulus_len(&self) -> usize {
         self.n_bits.as_usize_bytes_rounded_up()
     }
+
+    /// Exports the public modulus as big endian to the output array.
+    ///
+    /// The array should be of at least the size returned by
+    /// `public_modulus_len`.
+    pub fn export_public_modulus(&self, out: &mut [u8])
+                    -> Result<(), error::Unspecified> {
+        self.n.fill_be_bytes(out)
+    }
+
+    /// Return the public exponent.
+    pub fn public_exponent(&self) -> u64 {
+        self.e.value()
```

Comment:
Maybe it's better to output the exponent as a big-endian byte slice, like the modulus? That would be symmetric with `ring::signature::verify_rsa` and `export_public_modulus()`, and it is probably what all the callers need anyway.